src/utils/clients/discogs/formatters/album-search-response.formatter.test.ts

```diff
@@ -0,0 +1,61 @@
+import { albumSearchFormatter } from './album-search-response.formatter';
+
+it('should format the results', () => {
+  const results = albumSearchFormatter({
+    data: {
+      results: [
+        {
+          type: 'master',
+          id: 1,
+          master_id: 2,
+          title: 'title',
+          year: 2022,
+          thumb: 'images.com/thumb',
+          cover_image: 'images.com/cover',
+        },
+        {
+          type: 'release',
+          id: 1,
+          master_id: 2,
+          title: 'title',
+          year: 2022,
+          thumb: 'images.com/thumb',
+          cover_image: 'images.com/cover',
+        },
+      ],
+    },
+  });
```

Comment:
To make the test easier to digest, this could be split into a base case where you test the value mappings (e.g. for `master`), and a second case where you check only the props that change with the other type (e.g. checking `discogsId` for a release)

Reply:
Fixed in 29b8571